projects/epbs-with-inclusion-list.md

```diff
@@ -0,0 +1,90 @@
+
+*What problem is your project is solving? Why is it important and what area of the protocol will be affected?*
+
+Currently, in Ethereum, the [PBS (proposer builder seperation)](https://ethresear.ch/t/proposer-block-builder-separation-friendly-fee-market-designs/9725) is seen in out of protocol systems like flashbots where the existing validators outsource block building to external entities. These external entities are sophisticated actors who can capture MEV and produce a block with much higher value (or juice) as compared to validators running on normal hardware. This design relies on a very small set of trusted entities called Relays which are responsible for proposer builder communication. At present, more than 90% of blocks in the network are broadcasted by 10 active relays. This introduces centralized risks and also makes the network more vulnerable to censorship.
+
+As a result, [this](https://ethresear.ch/t/why-enshrine-proposer-builder-separation-a-viable-path-to-epbs/15710) post advocates for having an in-protocol (enshrined) implementation of PBS as it can make the whole process more decentralized, censorship resistant and trustless. Moreover, having the block building process in-protocol makes the builders more accountable for their actions and makes the process more transparent.
+
+## Project description
+
+*What is your proposed solution?*
+
+As enshrining PBS is a complex problem and multuple designs have been proposed for the same, the goal is to implement a PoC for the PTC (Payload Timliness Committee) design along with inclusion list.
+
+## Specification
+
+*How will you implement your solutions? Give details and more technical information on the project.*
+
+There are quite a few research articles available for ePBS and as mentioned above, multiple designs have been proposed. Potuz and Terrence from the Prysm team have been involved in the discussions and have voted to go with the [PTC (Payload Timliness Committee) design](https://ethresear.ch/t/payload-timeliness-committee-ptc-an-epbs-design/16054). It is an updated design of another design initial proposed by Vitalik called [Two-slot PBS](https://ethresear.ch/t/two-slot-proposer-builder-separation/10980).
+
+This requires changes in both the consensus as well as the execution layer. Potuz and Terrence have been working on the consensus specs and the plan is to fully understand them, review them and implement them in the Prysm consensus client. For execution layer, the specs (and an EIP) will be written at a later stage (and I hopefully plan to contribute to writing that as well) and the chosen client is Geth. The plan is to break down the whole implementation into different components and iteratively build out the required feature. Moreover, with ePBS, an inclusion list design proposed [here](https://ethresear.ch/t/no-free-lunch-a-new-inclusion-list-design/16389) will also be implemented for preventing censorship as both of them are complementary to each other.
```

Comment:
I see you elaborate on chosen clients here, you can still mention it above so it's clear. 

One thing to add here is maybe being bit more specific on what will be your part. It's a big project so I wonder whether you have some specific focus or you can just mention you will do general contributions at first.